pkg/querier/flamegraph.go

```diff
@@ -138,3 +144,24 @@ func ExportToFlamebearer(fg *querierv1.FlameGraph, profileType *typesv1.ProfileT
 		},
 	}
 }
+
+const maxNodesDefault = 2048
```

Comment:
I think this is the key part someone should review in this PR. I'm setting the number of nodes to 2048 by default, which can be low in some instances. There's also currently no way to override it on the frontend.